src/pages/hegel/reference/nothing/development.mdx

```diff
@@ -52,16 +53,61 @@ Hegel goes to add a link between the empty thinking (or empty intuiting) that fo
+
+[^1]: More context around the passage reads: "Spirit is not this power which, as the positive, avoids looking at the negative, as is the case when we say of something that it is nothing, or that it is false, and then, being done with it, go off on our own way on to something else. No, spirit is this power only by looking the negative in the face and lingering with it. This lingering is the magical power that converts it into being. - This power is the same as what in the preceding was called the subject, which, by giving existence to determinateness in its own element, sublates abstract immediacy, or, is only existing immediacy, and, as a result, is itself the true substance, is being, or, is the immediacy which does not have mediation external to itself but is itself this mediation" (Hegel 2018, 21/§32).
+
+### Houlgate
+
+Stephen Houlgate offers a detailed examination of the development of `nothing`. In contrast to Parmenides, whose idea of `nothing` is that of a contrast to `being`[^2], Houlgate points out that for Hegel, `nothing` is not to be regarded as a negative or absence of something but "a negative from which being is altogether absent ... the sheer and utter _nothing_, in which there is no trace of being whatsoever" (Houlgate 2022, 132).
+
+[^2]: "Parmenides conceives being from the outset in opposition to nothing: being _is_, whereas nothing is _not_. Furthermore, this opposition is absolute: being does not emerge from or pass into nothing, but remains always and only being. Being _is_, and it never proves to be anything other than being: it is changeless, 'uncreated and imperishable'" (Houlgate 2022, 138).
+
+Having established the absolute nothingness of `nothing`, Houlgate next points out that, by virtue of its purity as absolute nothingness, "nothing has an _immediacy_ to it: it is simply and immediately nothing, and nothing else besides" (Houlgate 2022, 132). Thus, this precise _immediacy_ proves `nothing` to be:
+
+> utterly indeterminate _being_: for simple immediacy, at this stage in speculative logic, is all that being is understood to be (Houlgate 2022, 132).
+
+This sense of immediacy is further stressed through the self-equality of `nothing`. This simple quality with itself at once means that `nothing` is _immediately itself_. Houlgate additionally refers to §88 of Hegel's _Encyclopaedia Logic_, where, Hegel writes: "And similarly, but conversely, nothing, as this immediate [term] that is equal to itself, is the same as being" (Hegel 1991, 141/§88). And so Houlgate concludes that, "just as pure being, by virtue of its indeterminacy, vanishes into nothing, so pure nothing, by virtue of its immediacy, vanishes into being" (Houlgate 2022, 143).
```

Comment:
I'm not convinced by this division that Houlgate makes between the indeterminacy of Being developing into Nothing, and the immediacy of Nothing developing into Being. But again, this is something I could do in my addition to the article.

Reply:
Yes, good points! Definitely something to delve deeper into. I think opening an open-ended discussion on this would be great. I've started one here quoting you but feel free to add to it (I'll also respond when I have a moment). https://github.com/systemphil/sphil/discussions

Reply:
Saucy!